### DISCUSSION TITLE
Corporate Plan

### DISCUSSION THREAD

**Discussion by ratson** [admob-plus/admob-plus#271] [General]
I maintain this project in my spare time, so issues raised here will only be handled when I have time.
For those who need better support, there are [Corporate Plans](https://ratson.name/fund-admob-plus/#corporate-plan) for subscription.

Any financial support would help me to keep improving the quality of this project!

**Comment by DonZheng** [comment]
Hi @ratson ,

Corporate Plan is a good idea. But considering most corporate-level plugin users may have dedicated dev/test teams, I reckon not many of them will subscribe to the plan.

To increase revenue of your plugin, It may be worthwhile adding a once-off plugin integration professional service to indivials or/and corporations. Basically individuals/companies pay you to help them make the plugin work in their project.

Just an idea. Wish it helps you get more funding.

Cheers :)

**Reply by ratson** [reply]
@DonZheng Thank you for the suggesting, I will consider that, but I really hope the docs could be improved good enough without my help.
